static/data/brothers_2020-08-17.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Please remove the old json when you update - we used to have a whole mess of different JSONs with different naming schemes that accumulated. better to just keep it in git history.